beyond-all-reason/teiserver
# Fix create balance when incomplete data

Fixes #314 

## Unit Test Steps
```
mix test --only balance_test
```

## Test Steps
Run local browser. Login as admin. Go to Admin > Matches > Show a match > Go to balance tab
It should have no errors
